bitex/formatters/binance.py

```diff
@@ -62,4 +62,12 @@ def open_orders(self):
 
     def wallet(self):
         """Return namedtuple with given data."""
-        raise NotImplementedError
+        data = self.json(parse_int=str, parse_float=str)['balances']
+        # balances = {d['asset']: d['free'] for d in data}
+        # nametuple can not first is number, and can not more then 255
+        balances = {}
+        for i in data:
+            available = i['free']
+            if float(available) > 0:
+                balances[i['asset']] = available
```

Comment:
no need to drop values of 0 and less. It might confuse people, since it gives the impression that their wallet isn't there

Reply:
Some exchange will give 0 coin(maybe many many coins), and other exchange not, so I think it's good to unify to same format.